inventory-fusion/src/main/scala/com/normation/inventory/provisioning/fusion/FusionReportUnmarshaller.scala

```diff
@@ -319,8 +318,7 @@ class FusionReportUnmarshaller(
              policyServerId <- boxFromOption(optText(agentXML \\ "POLICY_SERVER_UUID") ,"could not parse policy server id (tag POLICY_SERVER_UUID) from specific inventory")
           } yield {
             //cfkey is not mandatory
-            val cfKey = optText(agentXML \ "CFENGINE_KEY")
-
+            val cfKey = optText(agentXML \ "AGENT_KEY").orElse(optText(agentXML \ "CFENGINE_KEY"))
```

Comment:
would you mind also changing var name for "cfkey" to "agentKey" ? (or simply "key")